committee-code-of-conduct/README.md

```diff
@@ -15,9 +15,9 @@ The [charter](charter.md) defines the scope and governance of the Code of Conduc
 ## Members
 
 * Celeste Horgan (**[@celestehorgan](https://github.com/celestehorgan)**), CNCF
+* Carlos Tadeu Panato Jr. (**[@cpanato](https://github.com/cpanato)**), Mattermost
 * Karen Chu (**[@karenhchu](https://github.com/karenhchu)**), Microsoft
 * Nabarun Pal (**[@palnabarun](https://github.com/palnabarun)**), VMware
-* Tim Pepper (**[@tpepper](https://github.com/tpepper)**), VMware
 * Vallery Lancey (**[@vllry](https://github.com/vllry)**), Apple
 
```

Comment:
also update below in the `Term ends on August 7, 2022` section and add Tim to the `Emeritus Committee Members` section

Reply:
done, thanks for the review!